applications/dashboard/controllers/api/SourcemapsApiController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+use Garden\Web\Data;
+
+/**
+ * API Controller to deliver source map files for compiled static resources: js, css.
+ */
+class SourcemapsApiController extends AbstractApiController {
+
+    /**
+     * Get source map.
+     *
+     * @param string $path Relative path to source map file
+     * @return Data
+     */
+    public function get(string $path): Data {
```

Comment:
If we're sure we really don't want any kind of permission check on here, we should at least have an empty permission check (e.g. `$this->permission()`).

Reply:
Does empty permission check does nothing?
I hate it :-(
But ok, I'll do.

Reply:
@initvector There should not be a permission check there. Even if we're in private communities, these are static assets. They should be served regardless of the user's permission level if the configuration is setup for it.

Reply:
I understand opening up the permissions, but API endpoints are supposed to always have a permission check, even if it's empty. That convention is as old as API v2. I'm open to discussing if that should change, though.

Reply:
It should not apply to static asset controllers. That would mean `ThemesApiController` get endpoints, `LocalesApiController`, and `SourcemapsApiController`.

Do banned users not get served static assets served through an API? @tburry Thoughts?

Reply:
If we really want to keep it for consistency we could pass some constant. `ALWAYS_ALLOW` that would bypass any checks.